src/StatusQ/Components/StatusChatListAndCategories.qml

```diff
@@ -71,6 +71,7 @@ Item {
             spacing: 4
 
             StatusChatList {
+                objectName: "statusChatList"
```

Comment:
This name is not precise enough, it would clash with other `statusChatList`s if people name their's like that as well.
Try something like `chatListStatusChatList` or `statusChatListAndCategoriesChatList`. The name is very weird, but that component is very weird I guess, because we have a StatusChatList inside StatusChatListAndCategories

Reply:
At first, I understood that we can use containers to deconflict object names. Now I see this is not very practical and we don't do it. Fixed it.